### ISSUE TITLE
Build flags

### ISSUE THREAD

**Issue by permartinson** [chrismaltby/gb-studio#1591]
**Is your feature request related to a problem? Please describe.**
For creating different language versions, or adding debug-specific options, the ability to use build flags would be very helpful.

**Describe the solution you'd like**
I imagine a simple event very similar to the event group, with an additional dropdown to select one of the defined build flags. If the flag is set to true when compiling, the group will be included in the build. Otherwise it is ignored. The build flags could be defined in the Build Options section of the settings. A dropdown at the left of the build button could contain checkboxes to activate and deactivate the flags.

**Describe alternatives you've considered**
I have currently made a custom event for my project with a similar solution. But because I wanted to prioritize my time to focus on the game, I edit the flags directly in the .js file before compiling.

**Comment by maxoakland** [comment]
This would be useful for me. I already use build flags and just set them manually in an empty "Init" scene that I only use to initialize variables at the start of the game

**Comment by mxashlynn** [comment]
This feature could also be used to show the correct player controls for web builds vs ROM builds.

**Comment by maxoakland** [comment]
> This feature could also be used to show the correct player controls for web builds vs ROM builds.

Yeah! That's one of the ways I use them now. I just have to set them in a custom init scene. It would be really nice for it to be handled by the IDE
